css/index.css

```diff
@@ -335,9 +361,9 @@ img {
   align-items: center;
   width: 200px;
   height: 45px;
-  background-color: #17A2B8;
```

Comment:
I can't imagine you lower cased all these by hand, what code extension are you using that does this??

Reply:
So the funny thing is, when I look at the variables.less file, it's the same, and I know I didn't hand-touch the index.css, so I believe this is the consequence of my OS being Linux, which is ultra-picky about casing, and running less-watch-compiler. Something in the code is probably recompiling the CSS with a very specific idea about whether it should be upper case or lower case.